static/app/views/performance/utils.tsx

```diff
@@ -629,6 +635,11 @@ export function getIntervalLine(
         }) ?? {},
       data: [],
     });
+  } else if (format === IntervalLineFormat.REGRESSION_EPM) {
+    periodDividingLine.markLine.lineStyle = {
+      ...periodDividingLine.markLine.lineStyle,
+      color: theme.red300,
+    };
```

Comment:
<img width="1127" alt="Screenshot 2023-10-30 at 3 08 56 PM" src="https://github.com/getsentry/sentry/assets/22846452/f6db106b-df33-4c13-9922-f4b5ee6008ba">

Took a look locally and I noticed the styling for the baseline and regressed baseline were still being shown. I think we might want to reset the `additionalSeries = [currentPeriod]` or something to drop the baselines

Reply:
Great catch! Thanks, fixed this (and simplified a bunch as a result).